tzdata/Readme.txt

```diff
@@ -1,2 +1,63 @@
-The data files in this directory were obtained from the public IANA time zone database,
-http://www.iana.org/time-zones, version 2016d.
+README for the tz distribution
```

Comment:
Ah, saw that to late. I updated the readme to only reference the data source now.